src/Snowfire/Beautymail/BeautymailServiceProvider.php

```diff
@@ -31,9 +31,9 @@ public function boot()
         $this->loadViewsFrom(__DIR__.'/../../views', 'beautymail');
 
         try {
-            $this->app['mailer']->getSwiftMailer()->registerPlugin(new CssInlinerPlugin());
+            $this->app['mailer']->getSymfonyMailer()->registerPlugin(new CssInlinerPlugin());
```

Comment:
Unfortunately a little more is required to add support for Symfony Mailer, see #152.